modules/apps/foundation/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/sanitizer-configuration.xml

```diff
@@ -488,6 +488,25 @@ http://www.w3.org/TR/html401/struct/global.html
 	</tags-to-encode>
 
 	<tag-rules>
+		<tag name="picture" action="validate" />
+			<tag name="source" action="validate">
+			<attribute name="srcset">
+				<regexp-list>
+					<regexp name="onsiteURL"/>
+					<regexp name="offsiteURL"/>
+				</regexp-list>
+			</attribute>
+			<attribute name="media">
+				<regexp-list>
+					<regexp name="anything" />
+				</regexp-list>
+			</attribute>
+			<attribute name="type">
+				<regexp-list>
+					<regexp value="[a-zA-Z0-9]+/[a-zA-Z0-9]+" />
```

Comment:
@csierra Just out of curiosity, this is a mime type regex, correct? Mime types contain also dashes. Or is it any intent in leaving the dash outside? Thx!